src/assets/translations/pt/main.json

```diff
@@ -437,6 +438,25 @@
         "approve": "Aprovar",
         "revoke": "Revogar",
         "infinite": "Infinito ∞"
+      },
+      "staking": {
+        "delegate": "Delegar",
+        "begin_unbonding": "Retirar delegação.",
```

Comment:
There shouldn't be a dot here?
```suggestion
        "begin_unbonding": "Retirar delegação",
```

Reply:
Not blocker but def, bulls eye. fox props